dvc/command/experiments.py

```diff
@@ -164,7 +164,7 @@ def _collect_rows(
                         tree = "├─╨"
                     else:
                         tree = "│ ╟"
-                        parent = f" ({parent_rev[:7]})"
```

Comment:
I was trying to separate the `tree` logic, but the `new_checkpoint` state made it a bit hard. We could add this value to a column and later calculate the appropriate tree, but I don't think it's worth it considering this hacky alternate way as done here.